templates/java/mobile/tools/git-version.gradle

```diff
@@ -0,0 +1,33 @@
+buildscript {
```

Comment:
Some documentation about the git version would be great.

Reply:
https://proandroiddev.com/configuring-android-project-version-name-code-b168952f3323 I will add comments and this link in there